pyflakes/checker.py

```diff
@@ -1699,7 +1699,12 @@ def CALL(self, node):
                     with self._enter_annotation():
                         self.handleNode(keyword.value, node)
 
-        self.handleChildren(node)
+        if any(_is_typing(node.func, nt, self.scopeStack)
+                for nt in ("TypedDict", "NamedTuple")):
+            with self._enter_annotation(AnnotationState.NONE):
```

Comment:
this isn't correct, consider `TypedDict("X", {"y": "T"})`

Reply:
Indeed, my new commit should resolve this